edx_reverification_block/xblock/static/sass/main.scss

```diff
@@ -0,0 +1,150 @@
+    border-bottom: 1px solid $highlight-color;
+    border-right: 1px solid $highlight-color;
+    box-shadow: -1px 2px 3px $light-gray;
+}
+
+.hidden {
+    display: none
+}
+
+.reverify-clear-line {
+    clear: both;
+}
+
+.reverify-xblock {
+    .action {
+        font: 16px/1.3 "Open Sans","Helvetica Neue",Helvetica,Arial,sans-serif;
+        box-sizing: border-box;
+        cursor: pointer;
+        transition: color .25s ease-in-out, background .25s ease-in-out;
+        display: inline-block;
+        border: 1px solid #007aba;
+        border-radius: 4px;
+        padding: 10px 15px;
+        text-align: center;
+        text-decoration: none;
+        background-image: none;
+        box-shadow: none;
+        text-shadow: none;
+    }
+
+    .action:hover,
+    .action:active,
+    .action:focus {
+        background-color: #268dc4;
+        border: 1px solid #268dc4;
+        color: #fff;
+        text-decoration: none;
+        background-image: none;
+        box-shadow: none;
+        text-shadow: none;
+    }
+
+    .action-primary {
+        background-color: #007aba;
+        color: #fff;
+    }
+
+    .action-secondary {
+        border: 1px solid #fff;
+        background-color: transparent;
+        color: #fff;
+    }
+}
+
+.reverify-now-wrapper {
+    @include msg-box($red);
+
+    a {
+        text-decoration: none;
+        color: $light-blue;
+    }
+
+    .reverify-now-title {
+        font-weight: bold;
+        margin-bottom: 10px;
+    }
+
+    .reverify-skip-msg {
+        margin-right: 30px;
+    }
+
+    .reverify-now-button {
+        margin-top: 15px;
+        width: 100%;
+    }
+}
+
+.reverify-skip-confirm-wrapper {
+    padding: 20px 30px;
+    background-color: black;
+    border-left: 5px solid $red;
+
+
+    .reverify-skip-confirm {
+        color: white;
+    }
+
+    .reverify-skip-confirm-title {
+        font-weight: bold;
+        margin-bottom: 10px;
+    }
+
+    .reverify-skip-actions {
+        margin-top: 10px;
+    }
+
+    .reverify-skip-cancel-button {
+        padding: 10px 30px;
+        background-color: black;
+        border: 1px solid white;
+        color: white;
+        width: 100%;
+    }
+
+    .reverify-skip-confirm-button {
+        @extend .reverify-action-primary;
+        margin-top: 15px;
+        width: 100%;
+    }
+}
+
+@media screen and (min-width: 716px) {
+    .reverify-now-wrapper {
+        .reverify-now-content {
+            width: 80%;
+            position: relative;
+            float: left;
+        }
+
+        .reverify-now-button {
+            width: 20%;
+            float: right;
+        }
+    }
+
+    .reverify-skip-confirm-wrapper {
+        .reverify-skip-actions {
+            float: right;
+        }
+
+        .reverify-skip-cancel-button {
+            width: auto;
+            display: inline-block;
+        }
+
+        .reverify-skip-confirm-button {
+            width: auto;
+            display: inline-block;
+        }
```

Comment:
If both selectors have the same styles being attached could just use a comma to separate the selectors to remove repetition